jsons/serializers.py

```diff
@@ -158,28 +159,53 @@ def default_object_serializer(obj: object,
     private attributes (i.e. attributes that start with an underscore).
     :param strip_properties: if ``True`` the resulting dict will not contain
     values from @properties.
+    :param strip_class_variables: if ``True`` the resulting dict will not
+    contain values class variables.
```

Comment:
You forgot 'from'. :-)
```suggestion
    contain values from class variables.
```